Server/bin/OpenClubList.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs");
+
+function ReadClubFile()
+{
+    var jsonString = fs.readFileSync("./data/datafromclub.json", "utf8");
```

Comment:
Get this data relative to the script location using [`__dirname`](https://nodejs.org/docs/latest/api/modules.html#__dirname).